networkx/algorithms/graph_hashing.py

```diff
@@ -70,12 +70,12 @@ def weisfeiler_lehman_graph_hash(
         Number of neighbor aggregations to perform.
         Should be larger for larger graphs.
     digest_size: int, optional (default=16)
-        Size (in bits) of blake2b hash digest to use for hashing node labels.
+        Size (in bytes) of blake2b hash digest to use for hashing node labels.
 
     Returns
     -------
     h : string
-        Hexadecimal string corresponding to hash of the input graph.
+        Hexadecimal string corresponding to hash of the input graph (:code:`2 * digest_size` hex characters).
```

Comment:
minor tweaks to doc formatting (nx uses double-backticks for code) and more compact for line length. We could break the line instead... but this seemed more direct and no less clear.
```suggestion
        Hexadecimal string corresponding to hash of `G` (length ``2 * digest_size``).
```